src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1862,6 +1861,17 @@ MavlinkReceiver::handle_message_rc_channels_override(mavlink_message_t *msg)
 	rc.values[16] = man.chan17_raw;
 	rc.values[17] = man.chan18_raw;
 
+	// check how many channels are valid
+	for (int i = 17; i >= 0; i--) {
+		const bool ignore_field = rc.values[i] == UINT16_MAX ||
```

Comment:
Should the ignored values be zeroed before passing along to the rest of PX4? Might be slightly safer than sending something that could be interpreted as max.

Reply:
Plausible improvement. I can modify it.

Reply:
8da5ad6af5e01ea58b31a0e0c14c19e55bc4528a